crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -270,6 +270,9 @@ def ha_config_check
 
       def deployment_check
         ret = {}
+        # Only allow the upgrade for mariadb setup
+        db_node = ::Node.find("roles:database-server").first
```

Comment:
This is fine while we only allow one database proposal. Might be worth adding a TODO here to follow up after we decide on a database-migration strategy to check this is still adequate.

Reply:
good point